screens/MainScreen.js

```diff
@@ -0,0 +1,142 @@
+import React, {useEffect, useState} from 'react';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+import Icon from 'react-native-vector-icons/Ionicons';
+import MenuCard from '../Components/MenuCard';
+import {
+  Image,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+
+const MainScreen = props => {
+  const [data, setData] = useState([]);
+  const [searchTerm, setSearchTerm] = useState('');
+  const getFoodImages = async () => {
+    try {
+      const response = await fetch('https://burgers1.p.rapidapi.com/burgers', {
+        method: 'GET',
+        headers: {
+          'X-RapidAPI-Key':
+            'b13d4452e0msh022c443e270f56bp1d7113jsn7bdfb29892ae',
+          'X-RapidAPI-Host': 'burgers1.p.rapidapi.com',
+        },
+      });
+      const json = await response.json();
+      setData(json);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+  useEffect(() => {
+    getFoodImages();
+  }, []);
+  return (
+    <KeyboardAwareScrollView>
+      <View style={styles.screenContainer}>
+        <View style={styles.menuIconContainer}>
+          <Image
+            source={require('../assets/images/menu.png')}
+            style={styles.menuIconStyle}
+          />
+          <Icon
+            name="cart"
+            size={30}
+            color="#808080"
+            style={styles.cartIconStyle}
+          />
+        </View>
+
+        <Text style={styles.text}>Delicious food for you</Text>
+        <View style={styles.inputView}>
+          <Icon
+            name="search-outline"
+            size={25}
+            style={styles.alignIcon}
+            color="#000000"
+          />
+          <TextInput
+            style={styles.input}
+            placeholder="Search"
+            placeholderTextColor="#000000"
+            onChangeText={val => {
+              setSearchTerm(val);
+            }}
+          />
+        </View>
+        <>
+          <ScrollView horizontal={true} style={styles.galleryContainer}>
+            {data
+              .filter(val => {
+                if (searchTerm === '') {
```

Comment:
simplify the two logics returning the same value.
Fix the filter usage. filter only returns boolean